app/styles/_carousel-complex.scss

```diff
@@ -16,6 +16,12 @@ $carousel-item-size: 80px;
 .carousel-complex__navigation-text {
   border: none;
   padding-bottom: 10px;
+
+  a {
+    padding: .2em .5em;
+    background-color: #fff;
+    font-size: .9em;
```

Comment:
What about adding: 

    border: 1px solid #c74f0a;
    border-radius: 3px;
    position: relative;
    top: -1px;

And changing padding to `.1em .5em;`

Reply:
Yes, this would be another basic style update.